### ISSUE TITLE
convergence tests fails

### ISSUE THREAD

**Issue by mauro3** [pwl/DASSL.jl#7]
I tried to run the convergence test:
```
...v0.3/DASSL/test((HEAD detached at a17ab3f))  >> julia convergence.jl 
Warning: could not import Base.Text into Tk
INFO: Stepsize too small (h=3.637978807091713e-16 at t=0.0.
INFO: Stepsize too small (h=3.637978807091713e-16 at t=0.0.
ERROR: dimensions must match
 in - at array.jl:723
 in dasslTestConvergence at /home/mauro/.julia/v0.3/DASSL/test/convergence.jl:35
 in dasslDrawConvergence at /home/mauro/.julia/v0.3/DASSL/test/convergence.jl:59
 in include at ./boot.jl:245
 in include_from_node1 at loading.jl:128
 in process_options at ./client.jl:285
 in _start at ./client.jl:354
while loading /home/mauro/.julia/v0.3/DASSL/test/convergence.jl, in expression starting on line 114
```

**Comment by pwl** [comment]
Yeah, convergence tests are ancient and they aren't really a part of the test suite (I just wrote them as a demonstration).  I will take a look and either remove or update them.

**Comment by ChrisRackauckas** [comment]
Could you add the convergence tests to the standard testing? That would be nice to ensure. If anyone else wants to modify the code, you'll want the convergence tests as part of Travis so you know nothing messed up the algorithm's fundamentals.

**Comment by pwl** [comment]
I'm not sure if my convergence tests are up to the current standard.  Shouldn't we test the convergence using an external package anyway?

**Comment by ChrisRackauckas** [comment]
Probably, yes. We should get that up and running (I didn't do a DAE part of the convergence tests since I only wrapped Sundials for that, and I think we can trust Sundials).

What are the problems used for testing DAEs?

**Comment by pwl** [comment]
My tests just involved a simple exponential problem to catch the obvious errors:-).

> What are the problems used for testing DAEs?

That's a good question!  I think ODETestSuite implemented some test cases, but I'm not an expert on that.

**Comment by mauro3** [comment]
You mean `IVPTesteSuite`, but that is probably deprecated in favor of Chris's package: https://github.com/JuliaDiffEq/Roadmap/issues/4
